src/transformers/tokenization_utils_base.py

```diff
@@ -1862,6 +1862,12 @@ def convert_added_tokens(obj: Union[AddedToken, Any]):
             with open(special_tokens_map_file, encoding="utf-8") as special_tokens_map_handle:
                 special_tokens_map = json.load(special_tokens_map_handle)
             for key, value in special_tokens_map.items():
+                if key in kwargs and kwargs[key]:
+                    # This value has already been redefined by the kwargs
+                    # We keep this new value and ignore the one stored in the special_tokens_map_file
+
+                    continue
```

Comment:
I just want to draw attention to this part of the code which is the main change and which inverts the priority rules.